remote_attestation/java/src/com/google/oak/remote_attestation/Attestor.java

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.google.oak.remote_attestation;
+
+import com.google.common.base.VerifyException;
+import com.google.oak.remote_attestation.crypto.AeadEncryptor;
+import com.google.oak.remote_attestation.crypto.KeyNegotiator;
+import java.security.GeneralSecurityException;
+
+public class Attestor {
+    /**
+     * Performs remote attestation and key negotiation.
+     */
+    static public class UnattestedPeer {
```

Comment:
```suggestion
    public static class UnattestedPeer {
```
This is the more conventional ordering of the modifiers.

Reply:
Done